VideoWeb/VideoWeb/ClientApp/src/app/shared/error/error.component.ts

```diff
@@ -44,4 +54,14 @@ export class ErrorComponent implements OnDestroy {
         clearTimeout(this.returnTimeout);
         this.subscription.unsubscribe();
     }
+
+    private getErrorMessage(): boolean {
+        this.errorMessage = new SessionStorage<string>(this.ERROR_MESSAGE_KEY);
+        this.errorMessageText = this.errorMessage.get();
+        return this.errorMessageText !== null;
+    }
+
+    reconnect(): void {
+        this.location.back();
```

Comment:
Hi Lionel,
I don't think location.back is going to refresh the previous page.  I think we need to use the Router instead to route to either vho screen or the waiting room depending on the origin.

Reply:
updated to router.navigate